parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -69,18 +69,43 @@ configureSwapFile() {
 }
 
 configureEtcEnvironment() {
+    mkdir -p /etc/systemd/system.conf.d/
+    touch /etc/systemd/system.conf.d/proxy.conf
+    chmod 0644 /etc/systemd/system.conf.d/proxy.conf
```

Comment:
unrelated, but wondering if we can get away with properly hardening file perms all around eventually, if not already (0644 -> 0600)

Reply:
hah, yes excellent point. I just noticed this in one of my later commits

probably good follow up